.gitignore

```diff
@@ -60,6 +61,6 @@ android/keystores/debug.keystore
 
 # generated by bob
 lib/
-
+src/type.ts
```

Comment:
![Screenshot 2023-08-05 at 2 42 22 PM](https://github.com/numandev1/react-native-keys/assets/36044436/904f9c09-2d04-4c8a-98c0-211d1d5eb575)

sorry we have to keep this file because if do yarn on root then i am getting this error